server.go

```diff
@@ -316,6 +330,27 @@ out:
 			} else {
 				srvrLog.Errorf("Can't find peer to send message %v", receiverID)
 			}
+		case msg := <- s.paymentManager.chPaymentOut:
```

Comment:
Similarly, I think this fragment can be removed all together from the PoV of the latest state in master.